client/rpc/broadcast.go

```diff
@@ -1,17 +1,101 @@
 package rpc
 
 import (
+	"github.com/amolabs/tendermint-amo/crypto"
+	cmn "github.com/amolabs/tendermint-amo/libs/common"
 	ctypes "github.com/amolabs/tendermint-amo/rpc/core/types"
-	"github.com/amolabs/tendermint-amo/types"
 
 	"github.com/amolabs/amoabci/amo/operation"
 	atypes "github.com/amolabs/amoabci/amo/types"
 )
 
 // Transfer handles transfer transaction
-func Transfer(to types.Address, amount *atypes.Currency) (*ctypes.ResultBroadcastTxCommit, error) {
-	return RPCBroadcastTxCommit(MakeMessage(operation.TxTransfer, operation.Transfer{
+func Transfer(to crypto.Address, amount *atypes.Currency, sign bool) (*ctypes.ResultBroadcastTxCommit, error) {
```

Comment:
after we start to support signed transaction, signing a transaction should be mandatory. an unsigned transaction will be not an option. and this comment applies to other transactions: register, request, ...